src/storage/src/hummock/sstable/mod.rs

```diff
@@ -411,16 +417,16 @@ impl SstableMeta {
 
     pub fn encode_to(&self, buf: &mut Vec<u8>) {
         let start_offset = buf.len();
-        buf.put_u32_le(self.block_metas.len() as u32);
+        buf.put_u32_le(u32::try_from(self.block_metas.len()).unwrap());
```

Comment:
May extract the logic of `u32::try_from(..).unwrap()` to a common method named like `assert_u32`